.gitignore

```diff
@@ -0,0 +1,5 @@
+**/__pycache__
```

Comment:
Is this file necessary for ignoring temp file when generating the documentation? Or is it for your convenience to not commit those?

@ValentinS4t1qbit we never defined a `gitignore` file yet, it could be the occasion to merge one.

Reply:
@alexfleury-sb I believe pycache folders contain the compiled Python code (compiled to C or something, pyc files).  They should never be committed in this repo, it's stuff thats generated at runtime by Python when you run anything. I'm very fine with this change